core/lib/spree/testing_support/factories/order_factory.rb

```diff
@@ -18,10 +18,14 @@
       line_items_price BigDecimal.new(10)
     end
 
+    # TODO: Improve the name of this factory.
```

Comment:
When using a tool to parse TODOs it would help to actually have the name of the factory in the message.

```ruby
# TODO: Improve the name of the order_with_totals factory
```